#!/usr/bin/env python
PACKAGE = "exploration"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("last_goal_tolerance", double_t, 0, "", 1.0, 0.0, 10.0)
gen.add("log_newer_limit", double_t, 0, "", 10.0, 0.0, 600.0)
gen.add("duplicate_tolerance", double_t, 0, "", 1.5, 0, 10.0)
gen.add("newer_duplication_threshold", double_t, 0, "", 100.0, 0.0, 1000.0)

exit(gen.generate(PACKAGE, "exploration", "sensor_based_exploration_parameter_reconfigure"))